### PR TITLE
Source item webapi

### PR DESCRIPTION
**Pull request by sandermangel**
### Fixed Issues (if relevant)

1. magento/magento2#<issue_number>: Issue title
2. ...

### Manual testing scenarios

1. ...
2. ...

### Contribution checklist
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds on Travis CI are green)

### CODE REVIEW

File: app/code/Magento/InventoryApi/Test/Api/SourceItemRepository/GetListTest.php [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+use Magento\InventoryApi\Api\Data\SourceItemInterface;
+use Magento\TestFramework\Assert\AssertArrayContains;
+use Magento\TestFramework\TestCase\WebapiAbstract;
+
+/**
+ * @covers Magento\InventoryApi\Api\SourceItemRepositoryInterface
+ */
+class GetListTest extends WebapiAbstract
+{
+    /**#@+
```
Comment (larsroettig) [inline app/code/Magento/InventoryApi/Test/Api/SourceItemRepository/GetListTest.php]: phpcdoc removed the support

File: app/code/Magento/InventoryApi/Test/Api/SourceItemsSaveTest.php [hunk 2]
```diff
@@ -0,0 +1,183 @@
...
+use Magento\InventoryApi\Api\Data\SourceItemInterface;
+use Magento\TestFramework\Assert\AssertArrayContains;
+use Magento\TestFramework\TestCase\WebapiAbstract;
+
+/**
+ * @covers \Magento\InventoryApi\Api\SourceItemsSaveInterface
+ */
+class SourceItemsSaveTest extends WebapiAbstract
+{
+    /**#@+
```
Comment (larsroettig) [inline app/code/Magento/InventoryApi/Test/Api/SourceItemsSaveTest.php]: phpcdoc removed the support

File: app/code/Magento/InventoryApi/Test/Api/SourceItemRepository/GetListTest.php [hunk 3]
```diff
@@ -0,0 +1,93 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
```
Comment (larsroettig) [inline app/code/Magento/InventoryApi/Test/Api/SourceItemRepository/GetListTest.php]: Please add the in all Files thank you  

`declare(strict_types=1);`

**Comment by naydav** [comment]
Red integration test is related to
https://github.com/magento-engcom/msi/issues/194

**Pull request merged**